docs/UsersGuide/source/Introduction.rst

```diff
@@ -88,58 +88,58 @@ The UFS SRW Application has been designed so that any sufficiently up-to-date ma
 
 * C and C++ compilers compatible with the Fortran compiler
 
-   * gcc v9+, ifort v18+, and clang v9+ (macOS, native Apple clang or LLVM clang) have been tested
+   * gcc v9+, ifort v18+, and clang v9+ (macOS, native Apple clang, LLVM clang, GNU) have been tested
 
 * Python v3.6+, including prerequisite packages ``jinja2``, ``pyyaml`` and ``f90nml``
    
    * Python packages ``scipy``, ``matplotlib``, ``pygrib``, ``cartopy``, and ``pillow`` are required for users who would like to use the provided graphics scripts
 
 * Perl 5
 
-* git v1.8+
+* git v2.12+
 
-..
-   COMMENT: Should curl/wget/TIFF library also be required? These are listed as prerequisites for building HPC-Stack on generic MacOS/Linux 
+* curl 
+
+* wget
 
+..
+   COMMENT: Should curl/wget/TIFF library also be required? These are listed as prerequisites for building HPC-Stack on generic MacOS/Linux  | A: TIFF could be installed with the HPC-stack build; others are needed
 
 The following software is also required to run the SRW Application, but the :term:`HPC-Stack` (which contains the software libraries necessary for building and running the SRW App) can be configured to build these requirements:
 
-* CMake v3.15+
+* CMake v3.20+
 
 * MPI (MPICH, OpenMPI, or other implementation)
 
-   * Only **MPICH** can be built with HPC-Stack. Other options must be installed separately by the user. 
-
-* Software libraries
+   * Only **MPICH** or **OpenMPI** can be built with HPC-Stack. Other implementations must be installed separately by the user. 
 
-   * netCDF (C and Fortran libraries)
-   * HDF5 
-   * ESMF 8.2.0
-   * Jasper
-   * libJPG
-   * libPNG
-   * zlib
+..
+    COMMENT: All the software libraries are removed from the list, as they are built by the HPC stack.
 
 For MacOS systems, some additional software is needed. It is recommended that users install this software using the `Homebrew <https://brew.sh/>`__ package manager for MacOS:
 
+* brew install bash  
+
+  * OR:  brew upgrade bash 
 * brew install gcc@11
 * brew install cmake
 * brew install make
 * brew install wget
+* brew install lmod
```

Comment:
@natalie-perlin I thought lmod was recommended, not required. Is it required for MacOS?